src/components/MapView/Analyser/index.tsx

```diff
@@ -86,17 +119,69 @@ function Analyser({ classes }: AnalyserProps) {
   const adminBoundariesExtent = useMemo(() => {
     if (!boundaryLayerData)
       // not loaded yet. Should be loaded in MapView
-      return null;
+      return undefined;
```

Comment:
My rule of thumb is to keep undefined for the computer and use null whenever we expect something to be empty. But we can keep it like this. Just personal preference :)